pol-core/pol/irequest.cpp

```diff
@@ -38,22 +38,19 @@ namespace Core
 {
 void statrequest( Network::Client* client, u32 serial )
 {
-  if ( serial == client->chr->serial )
-  {
-    send_full_statmsg( client, client->chr );
-  }
-  else
+  auto chr = client->chr;
+  if ( serial == chr->serial )
   {
-    Mobile::Character* chr = find_character( serial );
-    if ( chr != nullptr )
-    {
-      if ( client->chr->is_visible_to_me( chr ) )
-      {
-        if ( client->chr->in_visual_range( chr ) )
-          send_short_statmsg( client, chr );
-      }
-    }
+    send_full_statmsg( client, chr );
+    return;
   }
+  Mobile::Character* bob = find_character( serial );
+  if ( !bob )
+    return;
+  if ( chr->is_visible_to_me( bob ) )
+    send_short_statmsg( client, bob );
+  if ( chr->has_party() )
+    chr->party()->send_stat_to( chr, bob );
```

Comment:
does this check if bob is in chr's party?

Reply:
Yes is_member( bob->serial) gets called in send_stat_to